dashboards/system_tables/sql/job_concurrency_comparison_fast.sql

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
Same as before, can we maintain one copy of "slow" / "fast" if they're the same SQL.

Reply:
done